drivers/crypto/Kconfig.esp32

```diff
@@ -30,4 +39,14 @@ config CRYPTO_ESP32_SHA_SESSIONS_MAX
 	  suspended/resumed by saving/restoring the hardware state.
 	  Increase this value if multiple SHA streams need to run concurrently.
 
+config ESP32_CRYPTO_AES_SESSIONS_MAX
+	int "Max ESP32 AES sessions"
+	default 4
```

Comment:
again, I'd limit the default number of allowed session and maybe provide the range of maximum sessions